### PR TITLE
Run3-alca248Y Modify the macros to examine momentum dependence of the correction factors

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Modify the macros to examine the momentum dependence of the correction factors

#### PR validation:

These macros are used to show the range of momenta used by the calibration code in extracting the correction factors

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalCalibAlgos/macros/CalibPlotProperties.C [hunk 1]
```diff
@@ -1316,6 +1327,7 @@ void PlotHist(const char *hisFileName,
   gStyle->SetOptTitle(0);
   gStyle->SetOptStat(1110);
 
+  bool dataMC = false;
```
[the remark this answers is not in the document]
Comment (bsunanda) [reply Calibration/HcalCalibAlgos/macros/CalibPlotProperties.C:1330]: Yes . Those histograms are created only for MC

[the remark this answers is not in the document]
Comment (bsunanda) [reply Calibration/HcalCalibAlgos/macros/CalibPlotProperties.C]: PlotHist may be a method in a class or a standalone function.  Sorry for the confusion

[the remark this answers is not in the document]
Comment (bsunanda) [reply Calibration/HcalCalibAlgos/macros/CalibPlotProperties.C]: PlotThisHist is used by 2 separate functions - one of which could be used for data as well as MC and the PlotHist method is applicable for MC only

**Comment by perrotta** [comment]
+alca

**Comment by bsunanda** [comment]
@antoniovilela, @sextonkennedy, @rappoccio Could you merge this PR?

**Comment by rappoccio** [comment]
+1

**Pull request merged**